packages/edit-post/src/store/actions.js

```diff
@@ -270,25 +249,26 @@ export function showBlockTypes( blockNames ) {
  * what Meta boxes are available in which location.
  *
  * @param {Object} metaBoxesPerLocation Meta boxes per location.
- *
- * @yield {Object} Action object.
  */
-export function* setAvailableMetaBoxesPerLocation( metaBoxesPerLocation ) {
-	yield {
+export const setAvailableMetaBoxesPerLocation = ( metaBoxesPerLocation ) => ( {
+	dispatch,
+} ) =>
+	dispatch( {
 		type: 'SET_META_BOXES_PER_LOCATIONS',
 		metaBoxesPerLocation,
-	};
-}
+	} );
 
 /**
- * Returns an action object used to request meta box update.
- *
- * @yield {Object} Action object.
+ * Returns a promise of an action object used to request meta box update.
```

Comment:
This should do it:

```suggestion
 * Requests a metabox update
```

Otherwise we end up writing C++ in plain english :-) Promise of an action that requests sounds like a lot of pointers and it's hard to unwind in my head. This action is used to request an update, let's say it directly.

Reply:
Or, with even less indirection, "do a metaboxes update" 🙂 When doing thunk migrations, I often rewrite action descriptions like this. Instead of saying like "returns an action used to signal that X is requested," just say "do X."

Reply:
👍